website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -269,39 +286,51 @@ helm install \
         --set autoupdate.enabled=true
     ```
 
-### Manual Update
+### Manual update
 
-* To manually update the Appsmith container image to the latest release, run the command: `kubectl rollout restart statefulset appsmith`
+To manually update the Appsmith container image to the latest release, run the command: 
 
-## Nginx ingress controller compression (Gzip)
+```bash
+kubectl rollout restart statefulset appsmith
+```
 
-In the default installation, the Nginx ingress controller will not have the data compression configured. Please follow the following steps to enable it.
+## Uninstall
```

Comment:
Let this be 'Uninstall Helm chart' and move it to the bottom, just before Troublshooting

Reply:
This section uninstalls the Appsmith installation and not the Helm chart. I added it here as it completes the docs flow.

Reply:
As discussed - I will add `Uninstall Appsmith` and move the section just before Troubleshooting.